x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/panel/base_snooze_panel.tsx

```diff
@@ -223,7 +223,9 @@ export const BaseSnoozePanel: React.FunctionComponent<BaseSnoozePanelProps> = ({
                           height: '36px',
                           // Replicate euiPanel--accent vs euiPanel--subdued
                           // Applying these classNames by themselves doesn't work due to a CSS-in-JS issue with EuiPanel
-                          color: isActive ? '#a8376a' : euiTheme.colors.subduedText,
+                          color: isActive
+                            ? euiTheme.colors.textAccent
+                            : euiTheme.colors.textSubdued,
                           backgroundColor: isActive ? 'rgba(240,78,152,0.2)' : euiTheme.colors.body,
```

Comment:
@mgadewoll Should we also replace `rgba(240,78,152,0.2)`?

Reply:
Maybe, yes. Does it need the transparency or can it be a solid color?

Just looking at the code, I'd think it can become a solid color as this is a button. And based on button definitions it should then probably use `euiTheme.colors.backgroundLightAccent`.

Reply:
cc @georgianaonoleata1904 @joana-cps

Reply:
I reviewed it with @georgianaonoleata1904 and it looks good to me in Amsterdam and Borealis 👍

Reply:
@joana-cps Not blocking but looking at the code only, I'm curious why we are using a <button> here instead of EuiButton and why we're passing a custom color. Something to consider for later.

Reply:
yes good point @andreadelrio, I didn't look into the code, I just did a visual review.
Something to look into @georgianaonoleata1904 @cnasikas.